CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Unreleased Changes
+------------------
+
+* Feature - Add support for inheritance (#80)
```

Comment:
I'd expand this just slightly:
`Add support for inheritance. Aws Record models can now be extended using standard ruby inheritance (#80).`

Reply:
Revised!